src/screens/Customers/CustomerMain.tsx

```diff
@@ -197,20 +201,20 @@ function CustomerMain(props: RouteComponentProps & UserProps) {
         <h1 className={classes.title}>Customers</h1>
       </div>
       <div className={classes.selectionHeader}>
-          <UserSearchBar onSearchChange={handleSearchChange} />
-          <FormControl>
-            <Select onChange={handleMenuSelect} multiple value={sortAndFilter} inputProps={{ 'aria-label': 'Without label' }}>
-              <ListSubheader>Sort By</ListSubheader>
-              <MenuItem value="NAME">{SortBy.NAME}</MenuItem>
-              <MenuItem value="METER">{SortBy.METER}</MenuItem>
-              <ListSubheader>Filter By</ListSubheader>
-              <MenuItem value="PAYMENT_STATUS">{FilterBy.PAYMENT_STATUS}</MenuItem>
-              <MenuItem value="METER_STATUS">{FilterBy.METER_STATUS}</MenuItem>
-              <MenuItem value="ACTIVE_STATUS">{FilterBy.ACTIVE_STATUS}</MenuItem>
-            </Select>
-            <div className={classes.rightAlign}><FormHelperText>Sort and Filter</FormHelperText></div>
-          </FormControl>
-        </div>
+        <UserSearchBar onSearchChange={handleSearchChange} />
```

Comment:
Sorry for the formatting diffs. I can reset this before I merge. The only relevant change is the `.slice()`  and TODO comment on line 91